po/tr.po

```diff
@@ -93,11 +92,12 @@ msgid "<loglevel>"
 msgstr "<günlük-düzeyi>"
 
 #: newsboat.cpp:81
-#, fuzzy
 msgid ""
 "write a log with a certain log level (valid values: 1 to 6, for user error, "
 "critical, error, warning, info, and debug respectively)"
 msgstr "günlük tutarken düzey belirle (geçerli değerler: 1 - 6)"
```

Comment:
Shouldn't this old translation been removed?

Reply:
I guess I forgot to remove it. Is it okay if I remove it next time?

Reply:
Thanks for confirming! I'll just remove it myself now, otherwise it'd get concatenated with the correct translation, which would be confusing.

Reply:
Thanks and sorry for being too stupid to properly read the diff. :-/